flow/workflows/cdc_flow.go

```diff
@@ -528,6 +530,7 @@ func CDCFlowWorkflow(
 		IsResync:        cfg.Resync,
 	})
 	if err != nil {
+		syncStatusToCatalog(ctx, workflow.GetLogger(ctx), protos.FlowStatus_STATUS_FAILED)
```

Comment:
This should not fail, for e.g. if catalog is down temporarily

Reply:
what should we do? log warning & continue? retry?

Reply:
we can do LogFlowError and retry as bulk of alerting/monitoring relies on this information being available